docs/serverless/rules/rules-ui-create.mdx

```diff
@@ -575,11 +575,9 @@ When configuring an ((esql)) rule's **<DocLink slug="/serverless/security/rules-
         false-positive alerts.
 
     1. **MITRE ATT&CK<sup>TM</sup> threats** (optional): Add relevant [MITRE](https://attack.mitre.org/) framework tactics, techniques, and subtechniques.
-    1. **Custom highlighted fields** (optional): Specify highlighted fields for personalized alert investigation flows. Fields with values are added to the <DocLink slug="/serverless/security/view-alert-details" section="investigation">Highlighted fields</DocLink> section within the alert details flyout. Fields without values aren't added. After you create the rule, you can find all custom highlighted fields in the About section of the rule details page.
+    1. **Custom highlighted fields** (optional): Specify highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the <DocLink slug="/serverless/security/advanced-settings" section="update-sec-indices">default ((elastic-sec)) indices</DocLink>. There is no limit to the number of fields that you can add. 
```

Comment:
Same here, on this page `Rule edit > About section` we only show fields that are available in selected index patterns for the rule.

Reply:
```suggestion
    1. **Custom highlighted fields** (optional): Specify highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the you selected for the rule's data source.
```